ui/shared/components/panel/variants/VariantClassify.jsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Button, Label } from 'semantic-ui-react'
+import { CopyToClipboard } from 'react-copy-to-clipboard'
+import { HttpRequestHelper } from 'shared/utils/httpRequestHelper'
+import { ColoredIcon } from 'shared/components/StyledComponents'
+import DataLoader from 'shared/components/DataLoader'
+import AcmgModal from '../acmg/AcmgModal'
+import { VerticalSpacer } from '../../Spacers'
+import PopupWithModal from '../../PopupWithModal'
+import { getVariantMainTranscript } from '../../../utils/constants'
+
+const CLINGEN_ALLELE_REGISTRY_URL = 'http://reg.genome.network/allele'
+const CLINGEN_VCI_URL = 'https://curation.clinicalgenome.org/select-variant'
+
+class ClinGenAlleleId extends React.PureComponent {
```

Comment:
this is no longer an accurately named component as it has the allele ID but also the actual VCI link

Reply:
I changed it to ClinGenVciLink and moved the component into a new file.